app/src/main/res/values/themes.xml

```diff
@@ -18,6 +18,14 @@
     <item name="android:windowAnimationStyle">@style/Animation.Activity</item>
   </style>
 
+  <style name="RevisionCardActivityTheme" parent="Theme.MaterialComponents.Light.NoActionBar.Bridge">
```

Comment:
Shouldn't this theme inherit from the one that the activity was using before (OppiaThemeWithoutActionBar)?

Reply:
I notice doing this makes status bar color green instead of orange, even after overriding colorPrimaryDark

Reply:
Hmm odd. I'm not keen on the disparity here, but debugging these styles can be really time consuming so I think we can live with this.